readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -118,7 +118,18 @@ def append_conf(self, **__):
                 MkDocsYAMLParseError.INVALID_DOCS_DIR_CONFIG,
             )
 
+        # if user puts an invalid `docs_dir` path raise an Exception
+        if user_docs_dir:
+            checkout_path = self.project.checkout_path(self.version.slug)
+            possible_path = os.path.join(checkout_path, user_docs_dir)
```

Comment:
We actually need to check if this path doesn't exists https://github.com/rtfd/readthedocs.org/blob/4fea14e86fe0aabffd73352402ddf8dbba39ff40/readthedocs/doc_builder/backends/mkdocs.py#L159-L162

Reply:
Oh! you meant `docs_path` I thought you meant the `docs_dir` which user adds in the yaml. okay I'll Update the PR.

Reply:
Just to be clear, it is the one the users put in the mkdocs.yml file

Reply:
But the docs dir is relative to the mkdocs.yml file, which can be located in other places not jus in the root

Reply:
@stsewd Thanks for the explanation. :+1: